modules/dists/CyclicDist.chpl

```diff
@@ -85,12 +96,12 @@ to the ID of the locale to which it is mapped.
 
     use CyclicDist;
 
-    const Space = {1..8, 1..8};
-    const D: domain(2) dmapped Cyclic(startIdx=Space.lowBound) = Space;
+    const Dist = new cyclicDist(startIdx=(1,1));
+    const D = {1..8, 1..8} dmapped Dist;
```

Comment:
Should `dmapped` be avoided in this example since it's unstable? (I noticed `blockDist` uses it as well)

Reply:
I considered that question independent of this PR since this PR didn't change that behavior.  I'm still hoping to replace `dmapped` with something by Thursday if we can get consensus on it.